tasks/zadanie_15.php

```diff
@@ -17,4 +17,21 @@
 
 $shops = $params[0]; // tej linijki nie ruszamy :)
 
-echo "Najlepsza oferta: Sklep C — 1190 zł";
\ No newline at end of file
+foreach ($shops as $store => $price) {
+  $shopsCount = count($shops);
```

Comment:
tutaj ogólnie bardzo dużo się dzieje :sweat_smile: 
Możnaby było spróbować to zrobić ciut inaczej, krótsza propozycja:

```suggestion
  $minPrice = null;
  $shopName = null;
  foreach ($shops as $storeInfo) {
      if ($minPrice === null || $minPrice > $storeInfo['price']){
           $minPrice = $storeInfo['price'];
           $shopName = $storeInfo['store'];
      }
  }
```